simple_db_migrate/__init__.py

```diff
@@ -10,8 +10,9 @@
 SIMPLE_DB_MIGRATE_VERSION = '3.0.1'
 
 # fixing print in non-utf8 terminals
+# #42: fix for python3 encoding
 if hasattr(sys.stdout, 'encoding') and sys.stdout.encoding is not None and sys.stdout.encoding.upper() != 'UTF-8':
-    sys.stdout = codecs.getwriter('utf-8')(sys.stdout)
+    sys.stdout = codecs.getwriter('utf-8')(sys.stdout.detach())
```

Comment:
Can you check if this also works for you? 
Then we can keep compatibility with python2.

```suggestion
    stdout = (sys.version_info >= (3, 1)) and sys.stdout.detach() or sys.stdout
    sys.stdout = codecs.getwriter('utf-8')(stdout)
```

Reply:
Oh, I haven't noticed that the library supports python2 as well. Thanks for the suggestion. The suggested code worked with python3.7 on Windows.